instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+
+const SearchBar = (props) => {
+  return (
+    <form>
+      <input type="text"
+      name="textInput"
+      placeholder="Search"
+      value={props.textInput}
```

Comment:
where do you need to put your handler so that the page doesn't freak out when you hit return?